src/components/Bakeries/Schedule/Schedule.scss

```diff
@@ -0,0 +1,13 @@
+.schedule {
+  width: 80%;
+  margin: 0 auto;
+  padding: 10% 0%;
+
+  .title {
+    font-size: 20px;
+    margin-bottom: -1%;
```

Comment:
is % of width or height?

Reply:
% of margin-bottom )

Reply:
Hm, I'm not sure you get it right :)
Let's say I want to set margin-bottom as 10% of block height, can I do that?

Reply:
Actually we don't need this part of code! It works without it in an inappropriate way. However, if we want to change it, we can actually use it outside the percent class .schedule.